docs/es/week04/04.md

```diff
@@ -0,0 +1,13 @@
+---
+lang: es
+lang-ref: ch.04
+title: Semana 4
+translation-date: 26 Mar 2020
+translator: Manuel Pinar-Molina
+---
+
+
+## Práctica
+
+Comenzamos con un breve resumen de álgebra lineal y luego extendemos el tema a convoluciones usando datos de audio como ejemplo. Se reiteran conceptos clave como localidad, estacionalidad y matriz de Toeplitz. Luego haremos una demostración del rendimiento de la convolución en el análisis de tono. Finalmente, hay una breve digresión sobre la dimensionalidad de los diferentes datos.
+
```

Comment:
Can you add the two parts for the lectures?

Reply:
The summaries for the two lectures are missing.
Also, you need to update the `_config.yml` file.